server.js

```diff
@@ -89,3 +92,64 @@ os_info_data = JSON.stringify(os_info_data)
 
 
 }
+
+const server = http.createServer((request, response) => {
+  // Console.log any error that occurs when the request was made
+  request.on('error', (err) => {
+    console.log('An error ocurred')
+  })
+
+  // Console.log any error that occurs when the response was made
+  response.on('error', (err) => {
+    console.log('An error ocurred')
+  })
+
+  // set request.url to const urlPath for fast access
+  const urlPath = request.url
+
+  // Check if the urlPath == '/'. If true, load the index.html page
+  if (urlPath == '/')
+  {
+    response.writeHead(200, {'Content-Type': 'text/html'})
+    const data = fs.readFileSync('./pages/index.html')
+
+    response.write(data)
+    response.end()
+  }
+
+  // Else check if urlPath == '/about.html'. If true, load the about.html page
+  else if (urlPath == '/about')
+  {
+    response.writeHead(200, {'Content-Type': 'text/html'})
+    const data = fs.readFileSync('./pages/about.html')
+
+    response.write(data)
+    response.end()
+  }
+
+  // Else check if urlPath == '/sys'. If true, load the your os info has been saved successfully back to the user
+  else if (urlPath == '/sys')
+  {
+    response.writeHead(201, {'Content-Type': 'text/plain'})
+
+    response.write('Your os info has been saved successfully')
+    getAndSetOsInfo()
```

Comment:
# The getAndSetOsInfo Returns a true or false value
Put this true or false in  a variable "const success = getAndSetOsInfo()"
if( success ) send a response with status code 201 
else send a response with status code 500